pom.xml

```diff
@@ -181,6 +182,18 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>${slf4j.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.servicemix.bundles</groupId>
+                <artifactId>org.apache.servicemix.bundles.hamcrest</artifactId>
+                <version>${harmcrest.version}</version>
+                <scope>runtime</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.geronimo.specs</groupId>
+                <artifactId>geronimo-atinject_1.0_spec</artifactId>
+                <version>${geronimo-atinject.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
You don't seem to use it so let' remove it to keep it as atomic as possible

Reply:
Done, thanks